card/spec/lib/card/reference_spec.rb

```diff
@@ -110,81 +110,81 @@
 
   it "update referencing content on rename junction card" do
     @ab = Card["A+B"] #linked to from X, included by Y
-    @ab.update_attributes! :name=>'Peanut+Butter', :update_referencers => true
+    @ab.update_attributes! name: 'Peanut+Butter', update_referencers: true
     @x = Card['X']
     expect(@x.content).to eq("[[A]] [[Peanut+Butter]] [[T]]")
   end
 
   it "update referencing content on rename junction card" do
     @ab = Card["A+B"] #linked to from X, included by Y
-    @ab.update_attributes! :name=>'Peanut+Butter', :update_referencers=>false
+    @ab.update_attributes! name: 'Peanut+Butter', update_referencers: false
     @x = Card['X']
     expect(@x.content).to eq("[[A]] [[A+B]] [[T]]")
   end
 
   it "template inclusion" do
-    cardtype = Card.create! :name=>"ColorType", :type=>'Cardtype', :content=>""
-    Card.create! :name=>"ColorType+*type+*structure", :content=>"{{+rgb}}"
-    green = Card.create! :name=>"green", :type=>'ColorType'
+    cardtype = Card.create! name: "ColorType", type: 'Cardtype', content: ""
+    Card.create! name: "ColorType+*type+*structure", content: "{{+rgb}}"
+    green = Card.create! name: "green", type: 'ColorType'
     rgb = newcard 'rgb'
-    green_rgb = Card.create! :name => "green+rgb", :content=>"#00ff00"
+    green_rgb = Card.create! name: "green+rgb", content: "#00ff00"
 
     expect(green.reload.includees.map(&:name)).to eq(["green+rgb"])
     expect(green_rgb.reload.includers.map(&:name)).to eq(['green'])
   end
 
   it "simple link" do
-    alpha = Card.create :name=>'alpha'
-    beta = Card.create :name=>'beta', :content=>"I link to [[alpha]]"
+    alpha = Card.create name: 'alpha'
+    beta = Card.create name: 'beta', content: "I link to [[alpha]]"
     expect(Card['alpha'].referencers.map(&:name)).to eq(['beta'])
     expect(Card['beta'].referees.map(&:name)).to eq(['alpha'])
   end
 
   it "link with spaces" do
-    alpha = Card.create! :name=>'alpha card'
-    beta =  Card.create! :name=>'beta card', :content=>"I link to [[alpha_card|ALPHA CARD]]"
+    alpha = Card.create! name: 'alpha card'
+    beta =  Card.create! name: 'beta card', content: "I link to [[alpha_card|ALPHA CARD]]"
     expect(Card['beta card'].referees.map(&:name)).to eq(['alpha card'])
     expect(Card['alpha card'].referencers.map(&:name)).to eq(['beta card'])
   end
 
 
   it "simple inclusion" do
-    alpha = Card.create :name=>'alpha'
-    beta = Card.create :name=>'beta', :content=>"I include to {{alpha}}"
+    alpha = Card.create name: 'alpha'
+    beta = Card.create name: 'beta', content: "I include to {{alpha}}"
```

Comment:
Useless assignment to variable - `beta`.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.